IntegrationTests/CombinedConfig_FPGA2/script/Makefile

```diff
@@ -0,0 +1,10 @@
+
+# MODIFY THESE LINES WHEN COPYING AND PASTING THIS MAKEFILE
+TOP_FUNCS=$(FIRMWARE)/TopFunctions/CombinedConfig_FPGA2
+MODULES=ProjectionCalculator_L1L2ABC ProjectionCalculator_L1L2DE ProjectionCalculator_L1L2F ProjectionCalculator_L1L2G ProjectionCalculator_L1L2HI ProjectionCalculator_L1L2JKL ProjectionCalculator_L2L3ABCD ProjectionCalculator_L3L4AB ProjectionCalculator_L3L4CD ProjectionCalculator_L5L6ABCD ProjectionCalculator_D1D2ABCD ProjectionCalculator_D3D4ABCD ProjectionCalculator_L2L3ABCD ProjectionCalculator_L1D1ABCD ProjectionCalculator_L1D1EFGH ProjectionCalculator_L2D1ABCD  MatchProcessor_L1PHIA MatchProcessor_L1PHIB MatchProcessor_L1PHIC MatchProcessor_L1PHID MatchProcessor_L1PHIE MatchProcessor_L1PHIF MatchProcessor_L1PHIG MatchProcessor_L1PHIH MatchProcessor_L2PHIA MatchProcessor_L2PHIB MatchProcessor_L2PHIC MatchProcessor_L2PHID MatchProcessor_L3PHIA MatchProcessor_L3PHIB MatchProcessor_L3PHIC MatchProcessor_L3PHID MatchProcessor_L4PHIA MatchProcessor_L4PHIB MatchProcessor_L4PHIC MatchProcessor_L4PHID MatchProcessor_L5PHIA MatchProcessor_L5PHIB MatchProcessor_L5PHIC MatchProcessor_L5PHID MatchProcessor_L6PHIA MatchProcessor_L6PHIB MatchProcessor_L6PHIC MatchProcessor_L6PHID MatchProcessor_D1PHIA MatchProcessor_D1PHIB MatchProcessor_D1PHIC MatchProcessor_D1PHID MatchProcessor_D2PHIA MatchProcessor_D2PHIB MatchProcessor_D2PHIC MatchProcessor_D2PHID MatchProcessor_D3PHIA MatchProcessor_D3PHIB MatchProcessor_D3PHIC MatchProcessor_D3PHID MatchProcessor_D4PHIA MatchProcessor_D4PHIB MatchProcessor_D4PHIC MatchProcessor_D4PHID MatchProcessor_D5PHIA MatchProcessor_D5PHIB MatchProcessor_D5PHIC MatchProcessor_D5PHID TrackBuilder_L1L2 TrackBuilder_L2L3 TrackBuilder_L3L4 TrackBuilder_L5L6 TrackBuilder_D1D2 TrackBuilder_D3D4 TrackBuilder_L1D1 TrackBuilder_L2D1  VMStubMERouterTop_L1PHIA VMStubMERouterTop_L1PHIB VMStubMERouterTop_L1PHIC VMStubMERouterTop_L1PHID VMStubMERouterTop_L1PHIE VMStubMERouterTop_L1PHIF VMStubMERouterTop_L1PHIG VMStubMERouterTop_L1PHIH VMStubMERouterTop_L2PHIA VMStubMERouterTop_L2PHIB VMStubMERouterTop_L2PHIC VMStubMERouterTop_L2PHID VMStubMERouterTop_L3PHIA VMStubMERouterTop_L3PHIB VMStubMERouterTop_L3PHIC VMStubMERouterTop_L3PHID VMStubMERouterTop_L4PHIA VMStubMERouterTop_L4PHIB VMStubMERouterTop_L4PHIC VMStubMERouterTop_L4PHID VMStubMERouterTop_L5PHIA VMStubMERouterTop_L5PHIB VMStubMERouterTop_L5PHIC VMStubMERouterTop_L5PHID VMStubMERouterTop_L6PHIA VMStubMERouterTop_L6PHIB VMStubMERouterTop_L6PHIC VMStubMERouterTop_L6PHID VMStubMERouterTop_D1PHIA VMStubMERouterTop_D1PHIB VMStubMERouterTop_D1PHIC VMStubMERouterTop_D1PHID VMStubMERouterTop_D2PHIA VMStubMERouterTop_D2PHIB VMStubMERouterTop_D2PHIC VMStubMERouterTop_D2PHID VMStubMERouterTop_D3PHIA VMStubMERouterTop_D3PHIB VMStubMERouterTop_D3PHIC VMStubMERouterTop_D3PHID VMStubMERouterTop_D4PHIA VMStubMERouterTop_D4PHIB VMStubMERouterTop_D4PHIC VMStubMERouterTop_D4PHID VMStubMERouterTop_D5PHIA VMStubMERouterTop_D5PHIB VMStubMERouterTop_D5PHIC VMStubMERouterTop_D5PHID
```

Comment:
@aryd `ProjectionCalculator_L2L3ABCD` is duplicated in the list of modules:
```suggestion
MODULES=ProjectionCalculator_L1L2ABC ProjectionCalculator_L1L2DE ProjectionCalculator_L1L2F ProjectionCalculator_L1L2G ProjectionCalculator_L1L2HI ProjectionCalculator_L1L2JKL ProjectionCalculator_L2L3ABCD ProjectionCalculator_L3L4AB ProjectionCalculator_L3L4CD ProjectionCalculator_L5L6ABCD ProjectionCalculator_D1D2ABCD ProjectionCalculator_D3D4ABCD ProjectionCalculator_L1D1ABCD ProjectionCalculator_L1D1EFGH ProjectionCalculator_L2D1ABCD  MatchProcessor_L1PHIA MatchProcessor_L1PHIB MatchProcessor_L1PHIC MatchProcessor_L1PHID MatchProcessor_L1PHIE MatchProcessor_L1PHIF MatchProcessor_L1PHIG MatchProcessor_L1PHIH MatchProcessor_L2PHIA MatchProcessor_L2PHIB MatchProcessor_L2PHIC MatchProcessor_L2PHID MatchProcessor_L3PHIA MatchProcessor_L3PHIB MatchProcessor_L3PHIC MatchProcessor_L3PHID MatchProcessor_L4PHIA MatchProcessor_L4PHIB MatchProcessor_L4PHIC MatchProcessor_L4PHID MatchProcessor_L5PHIA MatchProcessor_L5PHIB MatchProcessor_L5PHIC MatchProcessor_L5PHID MatchProcessor_L6PHIA MatchProcessor_L6PHIB MatchProcessor_L6PHIC MatchProcessor_L6PHID MatchProcessor_D1PHIA MatchProcessor_D1PHIB MatchProcessor_D1PHIC MatchProcessor_D1PHID MatchProcessor_D2PHIA MatchProcessor_D2PHIB MatchProcessor_D2PHIC MatchProcessor_D2PHID MatchProcessor_D3PHIA MatchProcessor_D3PHIB MatchProcessor_D3PHIC MatchProcessor_D3PHID MatchProcessor_D4PHIA MatchProcessor_D4PHIB MatchProcessor_D4PHIC MatchProcessor_D4PHID MatchProcessor_D5PHIA MatchProcessor_D5PHIB MatchProcessor_D5PHIC MatchProcessor_D5PHID TrackBuilder_L1L2 TrackBuilder_L2L3 TrackBuilder_L3L4 TrackBuilder_L5L6 TrackBuilder_D1D2 TrackBuilder_D3D4 TrackBuilder_L1D1 TrackBuilder_L2D1  VMStubMERouterTop_L1PHIA VMStubMERouterTop_L1PHIB VMStubMERouterTop_L1PHIC VMStubMERouterTop_L1PHID VMStubMERouterTop_L1PHIE VMStubMERouterTop_L1PHIF VMStubMERouterTop_L1PHIG VMStubMERouterTop_L1PHIH VMStubMERouterTop_L2PHIA VMStubMERouterTop_L2PHIB VMStubMERouterTop_L2PHIC VMStubMERouterTop_L2PHID VMStubMERouterTop_L3PHIA VMStubMERouterTop_L3PHIB VMStubMERouterTop_L3PHIC VMStubMERouterTop_L3PHID VMStubMERouterTop_L4PHIA VMStubMERouterTop_L4PHIB VMStubMERouterTop_L4PHIC VMStubMERouterTop_L4PHID VMStubMERouterTop_L5PHIA VMStubMERouterTop_L5PHIB VMStubMERouterTop_L5PHIC VMStubMERouterTop_L5PHID VMStubMERouterTop_L6PHIA VMStubMERouterTop_L6PHIB VMStubMERouterTop_L6PHIC VMStubMERouterTop_L6PHID VMStubMERouterTop_D1PHIA VMStubMERouterTop_D1PHIB VMStubMERouterTop_D1PHIC VMStubMERouterTop_D1PHID VMStubMERouterTop_D2PHIA VMStubMERouterTop_D2PHIB VMStubMERouterTop_D2PHIC VMStubMERouterTop_D2PHID VMStubMERouterTop_D3PHIA VMStubMERouterTop_D3PHIB VMStubMERouterTop_D3PHIC VMStubMERouterTop_D3PHID VMStubMERouterTop_D4PHIA VMStubMERouterTop_D4PHIB VMStubMERouterTop_D4PHIC VMStubMERouterTop_D4PHID VMStubMERouterTop_D5PHIA VMStubMERouterTop_D5PHIB VMStubMERouterTop_D5PHIC VMStubMERouterTop_D5PHID
```

Reply:
Fixed